saturday-afternoon/components/flights/Airlines.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { Fragment } from "react";
+import { Col, Row, Avatar } from "antd";
+import { createFragmentContainer, graphql } from "react-relay";
+import idx from "idx";
+
+import type { Airlines as AirlinesType } from "./__generated__/Airlines.graphql";
+
+type Props = {
+  data: AirlinesType
+};
+
+const Airlines = ({ data: { legs } }: Props) => (
+  <Fragment>
+    {legs &&
+      legs.map(leg => {
+        if (leg) {
+          const logoUrl = idx(leg, _ => _.airline.logoUrl);
```

Comment:
Is there a reason why this line does not take care about default value but the next one does?

Reply:
because logoUrl then has a fallback (basically works as default value)